client/X11/xf_keyboard.c

```diff
@@ -43,6 +43,9 @@
 #include <freerdp/log.h>
 #define TAG CLIENT_TAG("x11")
 
+static BOOL controlDown = FALSE;
```

Comment:
These should be part of `xfContext` (same is true for the existing ones, looks like a bug)